src/sentry/runner/commands/devservices.py

```diff
@@ -81,19 +81,36 @@ def up(project, exclude):
     if "kafka" in settings.SENTRY_EVENTSTREAM:
         pass
     elif "snuba" in settings.SENTRY_EVENTSTREAM:
-        click.secho(
-            "! Skipping kafka and zookeeper since your eventstream backend does not require it",
-            err=True,
-            fg="cyan",
-        )
-        exclude |= {"kafka", "zookeeper"}
+        if not settings.SENTRY_USE_RELAY:
```

Comment:
@RaduW These nested "ifs" make it quite hard to read/understand.
Could we perhaps have an `if settings.SENTRY_USE_RELAY` block on the outer level, and do the `exclude` manipulations there?

Reply:
I could but I prefer it the way it is written.